zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/Publisher.java

```diff
@@ -172,8 +175,16 @@ public static boolean publish(Collection collection, String email, CollectionRea
                         return false;
                     }
 
+                    Future<Boolean> imageFuture = publishImages(collection);
+
                     publishComplete = publishFilesToWebsite(collection, email, collectionReader);
 
+                    try {
+                        publishComplete &= imageFuture.get().booleanValue();
```

Comment:
Consider using `&&` instead of `&=`. As I understand using `&&` will not attempt to evaluate the right hand side if publishComplete is already false. i.e. if the files fail to publish, it won't bother waiting to see if the images publish.

https://docs.oracle.com/javase/specs/jls/se8/html/jls-15.html#jls-15.23

I am thinking of a scenario where a publish fails, and for some reason if the image publishing hangs.